app/react/Users/components/usergroups/specs/UserGroupSidePanel.spec.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * @jest-environment jsdom
+ */
+import 'mutationobserver-shim';
+import Immutable from 'immutable';
+import { SidePanel } from 'app/Layout';
+import {
+  UserGroupSidePanel,
+  UserGroupSidePanelProps,
+} from 'app/Users/components/usergroups/UserGroupSidePanel';
+import { renderConnectedMount } from 'app/Templates/specs/utils/renderConnected';
+import { ReactWrapper } from 'enzyme';
+import React from 'react';
+import MultiSelect from 'app/Forms/components/MultiSelect';
+
+describe('UserGroupSidePanel', () => {
+  const userGroup = {
+    _id: 'group1Id',
+    name: 'Group 1',
+    members: [
+      { _id: 'user1', username: 'martha perez' },
+      { _id: 'user2', username: 'ana johnson' },
+    ],
+  };
+  const defaultProps: UserGroupSidePanelProps = {
+    users: [
+      { _id: 'user1', username: 'martha perez' },
+      { _id: 'user2', username: 'ana johnson' },
+      { _id: 'user3', username: 'maria rodriguez' },
+      { _id: 'user4', username: 'john smith' },
+    ],
+    userGroup,
+    opened: true,
+    userGroups: [{ ...userGroup }, { _id: 'group2Id', name: 'Group 2', members: [] }],
+    closePanel: jasmine.createSpy('onClick'),
+    onSave: jasmine.createSpy('onSave'),
+    onDelete: jasmine.createSpy('onDelete'),
+  };
+
+  let component: ReactWrapper<React.Component['props'], React.Component['state'], React.Component>;
```

Comment:
Don't we know these typings for the component returned by `renderConnectedMount` function? Could making that function generics help with this problem?